src/client/components/PageLayout/Toolbar/Options/GeoOptions/Zoom/Zoom.tsx

```diff
@@ -1,15 +1,30 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 
+import { useAppDispatch } from 'client/store'
+import { GeoActions, useGeoMapOptions } from 'client/store/ui/geo'
 import Button, { ButtonSize, ButtonType } from 'client/components/Buttons/Button'
+import { mapController } from 'client/utils'
 
 const Zoom: React.FC = () => {
+  const dispatch = useAppDispatch()
+  const { minZoom, maxZoom } = useGeoMapOptions()
+
+  const setZoom = useCallback(
+    (_zoom: number) => {
+      if (_zoom >= minZoom && _zoom <= maxZoom) {
```

Comment:
Is it possible that we can have a situation like:
minZoom: 3
current zoom in map: 3.5
Then when lowering the zoom, it would be `setZoom(mapController.getMap().getZoom() - 1)` -> `setZoom(3.5 - 1)`
Since the value is lower than the min zoom, we wouldn't update it. 

I think in that case we should set the zoom to the min. Same goes for the max zoom.
What do you think?